test/unit/adversarial/helper.py

```diff
@@ -232,11 +235,11 @@ def generate_data_combinations(n=10):
         Z = c % K
         assert X + Y * K + Z * K * K == c_orig
         X_type = dist_type[X]
-        X = datas[X]
+        X = dataX[X]
         Y_type = dist_type[Y]
-        Y = datas[Y]
+        Y = dataY[Y]
         Z_type = dist_type[Z]
-        Z = datas[Z]
+        Z = dataY[Z]
```

Comment:
@adrinjalali I tried to change as little as possible here and maintain consistency with the estimator checks changes - does this make sense to you? Y is required to be one dimensional according to the estimator checks so I have added real one dimensional options, added the same dimensionality to Z.